src/components/Table/index.js

```diff
@@ -266,9 +266,7 @@ const Table = () => {
             accessor: 'solutions',
             disableSortBy: true,
             Cell: cellInfo => {
-              const url = cellInfo.row.original.premium
-                ? `${cellInfo.row.original.url}/`
-                : cellInfo.row.original.url;
+              const url = `https://leetcode.com/problems/${cellInfo.row.original.url}/`;
```

Comment:
Thanks for the update here - I'm trying to remember why I had this conditional in the first place.. so weird 😅